app/views/submissions/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% posts.map do |s| %>
+  <br />
+  <% if s.text.blank? %>
+    <%= link_to(s.title, s.url) %>
+  <% else %>
+    <%= s.limit_text %>
```

Comment:
I'm beginning to wonder if it'd be a better user experience if this were linked to the `submissions#show` view. 🤔

Reply:
I've been thinking that too. What do you think of requiring a title on text posts?

Reply:
I can dig it. Are you envisioning limiting a Title's length?